examples/multicam/rs-multicam.cpp

```diff
@@ -66,6 +66,11 @@ int main(int argc, char * argv[]) try
         app.show(render_frames);
     }
 
+    auto my_list = rs2::aus_get_counters_list();
```

Comment:
This is no good.
Override an existing example is nice for local debugging but, 
This should not be pushed.

Reply:
removed

Reply:
I still see it here..

Reply:
fixed